seaworthy/tests-core/containers/test_base.py

```diff
@@ -87,3 +87,22 @@ def test_default_clean(self):
         """By default, the ``clean`` method raises a NotImplementedError."""
         with self.assertRaises(NotImplementedError):
             self.base.clean()
+
+    def test_get_host_port(self):
+        """
+        We can get the host port mapping of a container.
+        """
+        self.base.create_kwargs = lambda: {'ports': {
+            '8080/tcp': ('127.0.0.1',),
+            '9090/tcp': ('127.0.0.1', '10701'),
+        }}
+        dh = self.make_helper()
+        self.base.create_and_start(dh, pull=False)
+
+        # We get a random high port number here.
+        random_host_port = self.base.get_host_port('8080/tcp')
+        self.assertGreater(int(random_host_port), 1024)
+        self.assertLess(int(random_host_port), 102400)
```

Comment:
I thought the highest port number was 65535?

Reply:
Probably. I was lazy about choosing a number.